Use better variable names and remove unwanted code blocks.

humblec opened the pull request:
Signed-off-by: hchiramm <hchiramm@redhat.com>

<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label
3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/testing.md
4. If you want *faster* PR reviews, read how: https://git.k8s.io/community/contributors/guide/pull-requests.md#best-practices-for-faster-reviews
5. Follow the instructions for writing a release note: https://git.k8s.io/community/contributors/guide/release-notes.md
6. If the PR is unfinished, see how to mark it: https://git.k8s.io/community/contributors/guide/pull-requests.md#marking-unfinished-pull-requests
-->

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
<!-- 
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
<!--  
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
-->
```release-note

```

humblec committed:
Use better variable names and remove unwanted code blocks.

Signed-off-by: hchiramm <hchiramm@redhat.com>

humblec commented:
/release-note-none

humblec commented:
/kind cleanup

humblec commented:
/assign @jdef 
/assign @caesarxuchao

liggitt commented on pkg/cloudprovider/providers/photon/photon.go:
```diff
@@ -474,16 +474,16 @@ func (pc *PCCloud) InstanceID(ctx context.Context, nodeName k8stypes.NodeName) (
+	}
+	// We assume only master need to get InstanceID of a node other than itself
+	ID, err := getInstanceID(pc, name)
+	if err != nil {
```
simplify to this?
```
if err != nil {
  klog.Errorf("Photon Cloud Provider: getInstanceID failed for InstanceID. Error[%v]", err)
}
return ID, err
```

humblec replied:
sgtm. Done.

liggitt reviewed (commented):
a couple nits. given https://github.com/kubernetes/kubernetes/pull/72178, do we are about little cleanups to this code?

humblec committed:
Simplify return from InstanceID func()

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

humblec commented on pkg/cloudprovider/providers/photon/photon.go:
```diff
@@ -474,16 +474,16 @@ func (pc *PCCloud) InstanceID(ctx context.Context, nodeName k8stypes.NodeName) (
 		return pc.localInstanceID, nil
+	}
+	// We assume only master need to get InstanceID of a node other than itself
+	ID, err := getInstanceID(pc, name)
```
fine to me. Done.

humblec commented on pkg/cloudprovider/providers/photon/photon.go:
```diff
@@ -544,7 +544,7 @@ func (pc *PCCloud) HasClusterID() bool {
 }
 
-// Attaches given virtual disk volume to the compute running kubelet.
+// AttacheDisk attaches given virtual disk volume to the compute running kubelet.
```
done.

humblec commented:
@liggitt thanks , addressed the comments. Yeah, this may be deprecated, but its going to take some time, till then its good to have it addressed in our source.

liggitt commented:
/lgtm
/approve

humblec commented:
/test pull-kubernetes-integration
